public/lib/column-defaults.js

```diff
@@ -34,30 +34,39 @@ import publicationLocationTemplate from "components/content-list-item/templates/
  *      colspan: number, // colspan of this field
  *      title: string, // title attribute contents for the column heading
  *      templateUrl: string // URL for the content-list-item template for this field
- *      isSortable: boolean // Can the column be sorted by clicking its header?
- *      sortField: string // The field to sort on, if different from `name`. Can be an item path, e.g. `a.nested.field`
+ *      isSortable: boolean = false // Can the column be sorted by clicking its header?
+ *      sortField?: string // The field to sort on, if different from `name`. Can be an item path, e.g. `a.nested.field`
+ *      defaultSortOrder?: 'asc' | 'desc' // The default sort order for the column on first click. Defaults to 'asc'
+ *      flipSortIconDirection: boolean = false // Flip the direction of the sort chevron relative to the sort order. Defaults to 'desc' -> ▼
  * }
  */
 
 var templateRoot = '/assets/components/content-list-item/templates/';
 
-const createSortTemplate = (sortField, labelHTML) => `
+const chevronUp = '&#9660;'
+const chevronDown = '&#9650;'
+
+const createSortTemplate = (sortField, labelHTML, flipSortIconDirection = false) => `
     <div ng-click="toggleSortState('${sortField}')" class="content-list-head__heading-sort-by">
       ${labelHTML}
       <span
         class="content-list-head__heading-sort-indicator"
         ng-class="{invisible: !getSortDirection('${sortField}')}"
         ng-switch="getSortDirection('${sortField}')">
-        <span ng-switch-when="desc">&#9660;</span>
-        <span ng-switch-when="asc">&#9650;</span>
+        <span ng-switch-when="desc">${flipSortIconDirection ? chevronDown : chevronUp}</span>
```

Comment:
A ternary in a switch is a bit complicated. Can this be simplified somehow? Or add a comment to explain why it's needed?

Reply:
Agreed – could farm this out to a selector in the controller, as with `getSortDirection()` above? 

Or better still, store these variables in the function body above and use the vars in the template?

Definitely a comment is warranted 👍

Reply:
@akash1810 as this has hopefully been addressed, I'll merge this now, but lmk if additional changes would make things clearer 👍

Reply:
👍 great work!